src/replica/Configuration.cc

```diff
@@ -416,10 +418,10 @@ DatabaseFamilyInfo Configuration::addDatabaseFamily(DatabaseFamilyInfo const& in
     if (info.overlap <= 0) errors += " overlap(<=0)";
     if (!errors.empty()) throw invalid_argument(_context(__func__) + errors);
     if (_connectionPtr != nullptr) {
-        _connectionPtr->executeInOwnTransaction([&](decltype(_connectionPtr) conn) {
-            conn->executeInsertQuery("config_database_family", info.name, info.replicationLevel,
-                                     info.numStripes, info.numSubStripes, info.overlap);
-        });
+        string const query = _g.insert("config_database_family", info.name, info.replicationLevel,
+                                       info.numStripes, info.numSubStripes, info.overlap);
```

Comment:
It's not super safe to assume the order of the columns in a table, passing column names should be safer.

Reply:
This technique is already available with the alternative generator:
```
QueryGenerator const g(conn);
std::string const query =
        g.insertPacked("table",
                       g.packIds("id", "timestamp", "name"),
                       g.packVals(Sql::NULL_, Sql::NOW, "John Smith"));
```
This will generate:
```
INSERT INTO `table` (`id`,`timestamp`,`name`) VALUES (NULL,NOW(),'John Smith')
```
```

Reply:
In principle, I could even add the overloaded version of this method to allow bulk insert of many rows:
```
QueryGenerator const g(conn);
std::vector<std::string> const packedValues = {
     g.packVals(Sql::NULL_, Sql::NOW, "John Smith"),
     g.packVals(Sql::NULL_, Sql::NOW, "Vera Rubin"),
     g.packVals(Sql::NULL_, Sql::NOW, "Igor Gaponenko")
};
std::string const query =
        g.insertPacked("table",
                       g.packIds("id", "timestamp", "name"),
                       packedValues);
```
This will result in generating the following statement:
```
INSERT INTO `table`
    (`id`,`timestamp`,`name`)
    VALUES (NULL,NOW(),'John Smith'),
           (NULL,NOW(),'Vera Rubin'), 
           (NULL,NOW(),'Igor Gaponenko')
```